docs/styles/style.css

```diff
@@ -0,0 +1,95 @@
+* {
+  margin: 0;
+}
+
+body {
+  font-size: 16px;
+  font-family: "Roboto", Arial, sans-serif;
+  line-height: 1.5; /* baseline height 24p x*/
+}
+
+p {
+  font-size: 1rem;
+  margin-bottom: 1.5rem;
+}
+
+p.lead {
+  font-size: 1.2rem;
+  line-height: 1.5rem;
+}
+
+li {
+  font-size: 1rem;
+}
+
+ul, ol {
+  margin-bottom: 1.5rem;
+}
+
+span {
+  font-size: 1rem;
+}
+
+h1, h2, h3, h4, h5 {
+  font-family: "Poppins", Arial, sans-serif;
+  font-weight: 700;
+  line-height: 3rem;
+}
+
+h1 {
+  font-size: 3rem;
+  padding-top: 0.25rem;
+  padding-bottom: 1.25rem;
+}
+
+h2 {
+  font-size: 2.5rem;
+  padding-top: 0.4375rem;
+  padding-bottom: 1.0625rem;
+}
+
+h3 {
+  font-size: 2rem;
+  padding-top: 0.625rem;
+  padding-bottom: 0.875rem;
+}
+
+h4 {
+  font-size: 1.5rem;
+}
+h5 {
+  font-size: 1rem;
+  margin: 0;
+}
+
+/* LAYOUT */
+
+.bio {
+  border-top: 1px solid lightgray;
+  padding-top: 1.5rem;
+}
+
+.bio__img {
+  height: 7.5rem;
+  border-radius: 100px;
+  float: left;
+  margin-right: 20px;
+}
+/* to hide and display colored guides for typo elements*/
+@media screen and (max-width: 900px) {
```

Comment:
You made now on big Media-Query. Thats ok and it's the best for the browser (speed, filesize, …)

The bigger a Project gets the more complicated it gets to have the code in one query. you would like to see all the definitions right where they modify something. then you would probably do something like:

```css

.y {
 padding: 1rem;
}

@media screen and (min-width: 720px) {
  .y {
   padding: 2rem;
  }
}

/* lllooaaaats of other code here */


.x {
 color: red;
}

@media screen and (min-width: 720px) {
  .x {
   color: blue;
  }
}


```